src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -134,6 +133,21 @@ export class FocusMonitor implements OnDestroy {
     this._windowFocusTimeoutId = setTimeout(() => this._windowFocused = false);
   }
 
+  /**
+   * Event listener for `focus` and 'blur' events on the document.
+   * Needs to be an arrow function in order to preserve the context when it gets bound.
+   */
+  private _documentFocusAndBlurListener = (event: FocusEvent) => {
+    const target = event.target as HTMLElement|null;
+    for (let el = target; el; el = el.parentElement) {
```

Comment:
Should we add a comment explaining that we need this for `checkChildren` support?

Reply:
Done.